src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -864,13 +840,41 @@ def build_sdfg_from_gtir(
         offset_provider_type: The definitions of offset providers used by the program node
 
     Returns:
-        An SDFG in the DaCe canonical form (simplified)
+        The SDFG as a result of the GTIR lowering, without applying any transformation.
```

Comment:
You should add to the documentation:
- That `PruneCasts` is run.
- The description of the `Returns` is a bit confusing as you apply transformation on the GTIR, but not on the SDFG.
    I would reformulate it to something like: "The SDFG generated through the lowering, without applying any post processing to it."

For further documentation you should also add that the SDFG follows the rules that are outlined in ADR18.

Reply:
Done.